doc/nrf/documentation/build.rst

```diff
@@ -27,6 +27,7 @@ Complete the following steps to install the required tools:
 
 1. If you have not done so already, install the |NCS| as described in :ref:`auto_installation` or :ref:`manual_installation`.
 #. Install or update all required :ref:`Python dependencies <additional_deps>`.
+#. Install the additional Python documentation dependencies by running the command ``pip3 install -U -r doc/requirements.txt`` in the :file:`ncs/nrf` folder.
```

Comment:
This kind of sounds like the dependencies are Python documentation. Would "Python dependencies for building the documentation" work? I'd probably also prefer having the command on its own so the click-to-copy function is available.

Also, we could instruct the user to leave the command line open in the previous step and just change the command here to account for it.
```suggestion
#. Install the additional Python dependencies for building the documentation by running the following command:

   .. code-block::
    
       pip3 install -U -r nrf/doc/requirements.txt
```
Should we account for Linux and MacOS users like the normal Python dependencies do?

Reply:
@gmarull , anything else to be documented for Linux/macOS users?

Reply:
it's really the same command

Reply:
We have the three different tabs (for the difference between command line window and terminal window, as well as the slightly different command in Linux) in the [Python dependencies](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation/installing.html#install-additional-python-dependencies) section. Since we link to that in the previous step, it makes sense to me to either make or acknowledge the difference in this step as well.

Reply:
I agree with @gmarull, especially given that, starting Ubuntu 23.04, you cannot use `pip install` anymore without using a virtual environment.

Here are the instructions:
https://docs.zephyrproject.org/latest/develop/getting_started/index.html#get-zephyr-and-install-python-dependencies

Reply:
Added instructions. @gmarull, please check.